src/main/java/net/sf/jabref/logic/preferences/LastFocusedTabPreferences.java

```diff
@@ -0,0 +1,50 @@
+/*  Copyright (C) 2003-2015 JabRef contributors.
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package net.sf.jabref.logic.preferences;
+
+import java.io.File;
+import java.util.Objects;
+
+import net.sf.jabref.JabRefPreferences;
+
+public class LastFocusedTabPreferences {
+
+    private final JabRefPreferences preferences;
+
+
+    public LastFocusedTabPreferences(JabRefPreferences preferences) {
+        this.preferences = Objects.requireNonNull(preferences);
+    }
+
+    public void setLastFocusedTab(File file) {
+        String filePath = "";
+        if (file != null) {
+            filePath = file.getAbsolutePath();
+        }
+
+        preferences.put(JabRefPreferences.LAST_FOCUSED, filePath);
+    }
+
+    public boolean hadLastFocus(File file) {
+        String lastFocusedDatabase = preferences.get(JabRefPreferences.LAST_FOCUSED);
+        if (file == null) {
+            return false;
```

Comment:
One last remark. ;)

I would use fail fast, that is, write the method as follows:

```
if(file == null) {
  return false;
}
String lastFocusedDatabase ...
// remaining code
``` 

This has the advantage that input checks do not have to be found somewhere in the code. Furthermore, you reduce the number of indention by 1 for the typical use case, you separate the typical use case from the failure case. And if/else should be used when both branches are equally likely - but typically, we have one case which is very likely and one which is very unlikely - and the code should reflect that as well.